README.md

```diff
@@ -94,25 +98,25 @@ make BUILD_TAGS='seccomp apparmor'
 | seccomp   | syscall filtering                  | libseccomp development library  |
 | selinux   | selinux process and mount labeling | <none>                          |
 | apparmor  | apparmor profile support           | libapparmor development library |
-### Validate Your cri Setup
+### Validate Your `cri` Setup
 A Kubernetes incubator project called [cri-tools](https://github.com/kubernetes-incubator/cri-tools)
-includes programs for exercising CRI implementations such as the `cri plugin`.
+includes programs for exercising CRI implementations such as the `cri` plugin.
 More importantly, cri-tools includes the program `critest` which is used for running
 [CRI Validation Testing](https://github.com/kubernetes/community/blob/master/contributors/devel/cri-validation.md).
 
-Run the CRI Validation test to validate your installation of `containerd` with `cri`:
+Run the CRI Validation test to validate your installation of `containerd` with `cri` built-in:
```

Comment:
/s/built-in/built in/  

In this case the dash isn't used because it's not the built-in plugin you are talking about it's the cri plugin that's built into containerd.

Reply:
Will do

Reply:
Done.